### PR TITLE
Upgrade PortX version 1.1.3

### PR DESCRIPTION
**Pull request by netsarang-bigseungho**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/19298)

### CODE REVIEW

File: manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+PackageIdentifier: NetSarangComputer.PortX
+PackageVersion: 1.1.3
+Publisher: NetSarangComputer
```
Comment (ItzLevvie) [inline manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml:3]:
```suggestion
Publisher: Support
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml (hunk 1 above)
Comment (NetSarangComputer) [reply manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml:3]: Why do we have to change the publishing company?

Comment (ItzLevvie) [reply manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml:3]: To match Control Panel as this is what Windows Package Manager uses to map the application in `winget list`

If you would like to change the Publisher to something else, please update your application with the changes you would like to make.

Comment (NetSarangComputer) [reply manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml:3]: Oh, thank you. I'll change it to Support and upload it.

Comment (NetSarangComputer) [reply manifests/n/NetSarangComputer/PortX/1.1.3/NetSarangComputer.PortX.yaml]: Changed and committed.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**